src/translations/ru.json

```diff
@@ -0,0 +1,5 @@
+{
+    "app.text": "Редактируйте <code>src/App.js</code> и сохраните, чтобы рестартануть. Теперь с {what}!",
+    "app.learn-react-link": "Изучай Реакт",
+    "app.choose-lang": "Выберите язык"
+}
```

Comment:
use .js files for transactions (not .json) - it would helps when project will grow (because at the .json files we can't comment lines, or slit translations to the separated objects)

Reply:
converted to .js